test-memory.js

```diff
@@ -0,0 +1,14 @@
+const heapdump = require('heapdump');
+const uncached = require('./index.js');
+
+for (let i = 0; i < 100000; i++) {
+  require('./fixture.js')();
```

Comment:
Indentation should happen with tabs. Files should also end in a newline. If you're not using it already, I suggest installing the `EditorConfig` plugin for your text editor ;).